File: Eventures.AndroidApp/app/src/main/java/eventures/data/Event.java
package eventures.data;

import android.os.Build;

import androidx.annotation.RequiresApi;

import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

public class Event {
    private int id;
    private String name;
    private String place;
    private String start;
    private String end;
    private String totalTickets;
    private String pricePerTicket;
    private EventuresUser owner;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }


    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }


    public String getPlace() {
        return place;
    }

    public void setPlace(String place) {
        this.place = place;
    }


    public String getStart() {
        return start;
    }

    public void setStart(String start) {
        this.start = start;
    }


    public String getEnd() {
        return end;
    }

    public void setEnd(String end) {
        this.end = end;
    }


    public String getTotalTickets() {
        return totalTickets;
    }

    public void setTotalTickets(String totalTickets) {
        this.totalTickets = totalTickets;
    }


    public String getPricePerTicket() {
        return pricePerTicket;
    }

    public void setPricePerTicket(String pricePerTicket) {
        this.pricePerTicket = pricePerTicket;
    }


    public EventuresUser getOwner() {
        return owner;
    }
}

File: Eventures.DesktopApp/Data/Event.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace Eventures.DesktopApp.Data
{
    public class Event
    {
        public int Id { get; set; }

        public string Name { get; set; }
        public string Place { get; set; }
        public string Start { get; set; }
        public string End { get; set; }
        public int TotalTickets { get; set; }

        [Column(TypeName = "decimal(12,3)")]
        public decimal PricePerTicket { get; set; }

        public EventuresUser Owner { get; set; }
    }
}
